tests/alpha.model.tests/src/alpha/model/tests/util/FaceLatticeTest.xtend

```diff
@@ -461,4 +466,26 @@ class FaceLatticeTest {
 		
 		assertVerticesExist(lattice, #[0,1,4], #[0,1,2,3], #[0,3,4], #[1,2,4], #[2,3,4])
 	}
+	
+	@Test
+	def testThickEquality_1() {
+		val lattice = makeLattice("[N]->{[i,j]: 0<=i<2 and -N+10<j<N}")
+		val dim = lattice.rootInfo.dimensionality
+		
+		assertEquals(dim, 1)
+	}
+	
+	@Test
+	def testThickEquality_2() {
+		val lattice = makeLattice("[N]->{[i,j,k]: 0<=k<=-N+i+j and k<=2N-2i+j and -5+2N+i-2j<=k<=2N+i-2j}")
+		val dim = lattice.rootInfo.dimensionality
+		assertEquals(dim, 2)
+		
+		lattice.lattice.forEach[l | println(l.toString)]
```

Comment:
`println` can be commented out or removed

Reply:
Fixed!